doc/source/development/contributing_codebase.rst

```diff
@@ -828,3 +835,10 @@ This will put the text *New in version 1.1.0* wherever you put the sphinx
 directive. This should also be put in the docstring when adding a new function
 or method (`example <https://github.com/pandas-dev/pandas/blob/v0.20.2/pandas/core/frame.py#L1495>`__)
 or a new keyword argument (`example <https://github.com/pandas-dev/pandas/blob/v0.20.2/pandas/core/generic.py#L568>`__).
+
+If your code is a bugfix, add your entry to the relevant bugfix section. Avoid
```

Comment:
I find this a bit confusing. This section starts talking about the whatsnew, then the previous paragraph is about changes to an object docstring, and seems like here again we're talking about the whatsnew file again. I found this confusing when reading, seems like this is talking about a bugfix section and `Other` section in a docstring. Can we move this immediately after the other block you added?

Reply:
Thanks - agreed.